charts/website/templates/deployment.yaml

```diff
@@ -33,6 +33,130 @@ spec:
+          # get vars from: https://github.com/waldo-vision/waldo/blob/master/apps/web/.env.example
+          env:
+            # discord
+            - name: "DISCORD_CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-id
+                  name: {{ .Release.Name }}-discord
+            - name: "DISCORD_CLIENT_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-secret
+                  name: {{ .Release.Name }}-discord
+
+            # google
+            - name: "GOOGLE_CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-id
+                  name: {{ .Release.Name }}-google
+            - name: "GOOGLE_CLIENT_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-secret
+                  name: {{ .Release.Name }}-google
+            - name: "YOUTUBE_API_KEY"
+              valueFrom:
+                secretKeyRef:
+                  key:  yt-api-key
+                  name: {{ .Release.Name }}-google
+
+            # github
+            - name: "GITHUB_CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-id
+                  name: {{ .Release.Name }}-github
+            - name: "GITHUB_CLIENT_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-secret
+                  name: {{ .Release.Name }}-github
+
+            # battle.net
+            - name: "BTLNET_CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-id
+                  name: {{ .Release.Name }}-btlnet
+            - name: "BTLNET_CLIENT_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-secret
+                  name: {{ .Release.Name }}-btlnet
+
+            # twitch
+            - name: "TWITCH_CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-id
+                  name: {{ .Release.Name }}-twitch
+            - name: "TWITCH_CLIENT_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-secret
+                  name: {{ .Release.Name }}-twitch
+
+            # facebook
+            - name: "FB_CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-id
+                  name: {{ .Release.Name }}-facebook
+            - name: "FB_CLIENT_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  client-secret
+                  name: {{ .Release.Name }}-facebook
+
+            # cloudflare
+            - name: "CLOUDFLARE_TURNSTILE_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  key:  turnstile-secret
+                  name: {{ .Release.Name }}-cloudflare
+            - name: "CLOUDFLARE_TURNSTILE_SITE_KEY"
```

Comment:
Needs to be changed to NEXT_PUBLIC_CLOUDFLARE_TURNSTILE_SITE_KEY

Reply:
k, i'll do that in the next release, this is still using 0.0.1-alpha.2 which doesn't have that

Reply:
kk